src/pip/_internal/utils/appdirs.py

```diff
@@ -28,11 +28,6 @@ def user_config_dir(appname, roaming=True):
     return _appdirs.user_config_dir(appname, appauthor=False, roaming=roaming)
 
 
-def user_data_dir(appname, roaming=False):
-    # type: (str, bool) -> str
-    return _appdirs.user_data_dir(appname, appauthor=False, roaming=roaming)
```

Comment:
This changes the behavior on Windows - now pip will be looking for `%APPDATA%\pip\pip\pip.ini`, since the default `appauthor=None` causes appdirs to use `appname` as `appauthor` and include it as a part of the path.

Reply:
@chrahunt was your comment intended for this pull request ?
As far as I can see, pip code never uses `pip._internal.utils.appdirs.user_data_dir` which what this PR removes.

Reply:
It's been so long, but if no one has filed an issue then I'm sure it is OK. :)